build.gradle

```diff
@@ -12,6 +12,8 @@ allprojects {
         implementation group: 'com.fasterxml.jackson.core', name: 'jackson-annotations', version: jacksonVersion
         implementation group: 'com.fasterxml.jackson.datatype', name: 'jackson-datatype-jsr310', version: jacksonVersion
         implementation group: 'org.apache.commons', name: 'commons-lang3', version: apacheCommon
+        implementation group: 'org.hibernate', name: 'hibernate-core', version: hibernateVersion
+        implementation group: 'org.hibernate', name: 'hibernate-entitymanager', version: hibernateVersion

```

Comment:
You are not allowed to use Hibernate, as far as I know

Reply:
its for importing from `javax.persistence`